Sources_v3/Workers/ChannelUpdater.swift

```diff
@@ -86,4 +86,28 @@ class ChannelUpdater<ExtraData: ExtraDataTypes>: Worker {
             completion?($0.error)
         }
     }
+    
+    /// Add users to the channel as members.
+    /// - Parameters:
+    ///   - cid: The Id of the channel where you want to add the users.
+    ///   - users: User Ids to add to the channel.
+    ///   - completion: The completion. Will be called on a **callbackQueue** when the network request is finished.
```

Comment:
There's nothing like a callback queue in the updater. Maybe some copy&paste leftover?